docs/articles/licensing/gplv3/gplv3-blacklist.rst

```diff
@@ -0,0 +1,86 @@
+GPLv3 licensed production software Vs automotive industry
+---------------------------------------------------------
+
+Modern cars have around 100 million lines of code running on them,12 with 70% 
+of that code being in the head unit. Complexity is a non-trivial issue in 
+automotive software design. In addition to being complex, cars can be dangerous.
+
+Locking the software to the hardware by signing the original software image
+with a cryptographic key so that only an image provided by the supplier will 
+boot or install – is a common practice in embedded devices.
+This process of signing software images – so only images with the right key 
+will boot or install effectively prevents a user from modifying the software
+on the device since they have no access to the key needed to allow their 
+modified version to boot or install.
```

Comment:
Would you review my proposed changes above?

Reply:
Done!

Reply:
Late reply, I would disagree to the premise @gordanm. I don't think we are writing for any layman and hence shouldn't dumb it down too much. As has already been discussed though, we should only describe the problem as objective as possible and our proposal for a solution to it.

Reply:
@frznlogic fair point. Layman might be streched but it might be someone new to the field like the case you are a part of right now. It should be easily understandable and readable, that is the main point :).